### DISCUSSION TITLE
咋办，不搞2FA会不会不让用GitHub了……

### DISCUSSION THREAD

**Discussion by SqrtTan** [Meloong-Git/PCL#2780] [讨论]
\*\*GitHub

**Comment by zkitefly** [comment]
> \*\*GitHub

弄 2FA 有什么难吗

**Reply by hejiehao** [reply]
当然难
每次我都得找我那失踪的手机

**Comment by MoYuan-CN** [comment]
那是**2FA**（悲
用 Authenticator 吧，浏览器插件的类型
~~不过你想在脱离这浏览器的情况下过验证就有点麻烦了~~

**Reply by zkitefly** [reply]
> 那是**2FA**
> 用 Authenticator 吧，浏览器插件的类型
> ~~不过你想在脱离这浏览器的情况下过验证就有点麻烦了~~

抱歉，当时没看清输错了（

**Comment by SqrtTan** [comment]
成功加了2FA滚回来了
~本人没手机只能用模拟器了~

**Reply by Deep-Dark-Forest** [reply]
……~~如果你有外国手机号也行~~

**Reply by SqrtTan** [reply]
这**GitHub没有+86
~我祝它总部被炸~

**Reply by MoYuan-CN** [reply]
额 是的 所以我也才找了半天的攻略找到那个插件

**Comment by SuoHouDaiShi** [comment]
此时一位尊贵的 Google Authenticator 用户路过
<del>然后忘记开梯子打开 Authenticator 一次性 PIN 全没了</del>

**Reply by MoYuan-CN** [reply]
啊这...

**Reply by ghost** [reply]
此时一位高贵的 Microsoft Authenticator 用户路过
不需要开梯子，但2FA总是验证失败，最后打开 GitHub APP 授权登录

**Comment by WorldHim** [comment]
看到一个可自部署的网页端 2FA 管理 [Bubka/2FAuth](https://github.com/Bubka/2FAuth)

**Comment by FQO3** [comment]
可以去手机搞个神锁离线版吧？我反正误打误撞过了2FA

**Comment by g1thubhack3r** [comment]
不建议在这里讨论GitHub，这是PCL2的仓库

**Reply by MoYuan-CN** [reply]
没事... 这是**讨论**频道

**Comment by MFn233** [comment]
可以用2fast，微软商店的软件

**Comment by Deep-Dark-Forest** [comment]


**Reply by burningtnt** [reply]
他大概率只是想用 ** 代替国粹

**Reply by hejiehao** [reply]
![image](https://github.com/Hex-Dragon/PCL2/assets/98140773/159931e9-011e-4d2b-83ff-6ea6a0ce470b)
~~点这个也行~~

**Comment by ghost** [comment]
> Authenticator



> **GitHub

事实上你不会被限制登录，你只会被限制提交代码和Issuse
至于为什么，因为 Microsoft 的 npm 曾经因为没有2FA 有个盗号狗盗了开发者账号乱提交代码致使npm上游项目混乱直接影响到了下游项目

**Comment by MoYuan-CN** [comment]
最新发现：阿里云 App 有一个虚拟 MFA，可以用于 2FA（MFA）验证
App 主页搜索 `虚拟 MFA` 即可，就算不是阿里云的也可以添加

**Reply by Ad-closeNN** [reply]
~~10月的话题现在来考古了是吧~~
